components/map/events.md

```diff
@@ -120,9 +127,14 @@ The `OnClick` event fires when the user clicks on the map. Its `EventCallback<Ma
 
 ## OnMarkerClick
 
-The `OnMarkerClick` event fires when the user clicks on a marker. Its `EventCallback<MapMarkerClickEventArgs>` gives:
-* `MapMarkerClickEventArgs.DataItem` - provides the data item (object) of the bound marker.
-* `MapMarkerClickEventArgs.EventArgs` - provides the native DOM event (browser event).
+The `OnMarkerClick` event fires when the user clicks on a marker. The `OnMarkerClick` event handler receives an argument of type `MapMarkerClickEventArgs`, which exposes the following properties:
```

Comment:
Usually, it's "click something" rather than "click on something". The only exception throughout this article would be the `OnClick` section, where "click on the map" implies that the user clicks anywhere on/within the map. And if you decide to go with `Map` as the name of the component, you can also use `clicks the Map` (you're using Map below). 

In all other instances, (i.e., "clicks a marker", "clicks a shape") would be the better choice. Refer to the [Microsoft Manual of Style](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/mouse-mouse-interaction-terms), since it seems we haven't included this as an explicit point in our style guide (although, it's the prevalent form in our examples).

I would also suggest (as per the MMoS recs) to user more specific verbs. For example, if the same event fires when the user taps something on a touch-enabled device, you might want to use `clicks or taps X`.